admin_guide/pruning_resources.adoc

```diff
@@ -207,21 +212,46 @@ prune:
 - the `--keep-tag-revisions` most recent items in
  `*stream.status.tags[].items*`.
 
+* Remove any image "managed by {product-title}" (i.e., images with the annotation
+`*openshift.io/image.managed*`) that is exceeding the smallest xref:limits.adoc#image-limits[limit]
+defined in the same project and is not currently referenced by:
+- any running pods.
+- any pending pods.
+- any replication controllers.
+- any deployment configurations.
+- any build configurations.
+- any builds.
+
 * There is no support for pruning from external registries.
 
 * When an image is pruned, all references to the image are removed from all
 image streams that have a reference to the image in `*status.tags*`.
 
 * Image layers that are no longer referenced by any images are removed as well.
 
+[NOTE]
+====
+`--prune-over-size-limit` cannot be combined with `--keep-tag-revisions` nor
+`--keep-younger-than` flags. Doing so, will return an information that this
+operation is not allowed.
+====
+
 To see what a pruning operation would delete:
 
 ----
 $ oadm prune images --keep-tag-revisions=3 --keep-younger-than=60m
 ----
 
+----
```

Comment:
Could you provide some description here as well? Or perhaps just merge it with the above command.